aptos-move/framework/aptos-framework/sources/vesting.move

```diff
@@ -1461,40 +1459,6 @@ module aptos_framework::vesting {
         setup_vesting_contract(admin, &vector[@1, @2], &vector[1], admin_address, 0);
     }
 
-    #[test(aptos_framework = @0x1, admin = @0x123)]
-    #[expected_failure(abort_code = 0x60001, location = aptos_framework::aptos_account)]
-    public entry fun test_create_vesting_contract_with_invalid_withdrawal_address_should_fail(
```

Comment:
Are there new tests that can replace these in the new paradigm? I'm not a fan of deleting tests in unrelated areas without proper replacements

Reply:
it's all about account. I can recover that by removing the expected error.

Reply:
the old tests just tests the registration would fail. Now it never fails... I don't think tests cannot be removed for this case.